src/modules/transaction/components/TransactionRow/schemas.css

```diff
@@ -1,11 +1,14 @@
 .container {
+  color: transparent;
+  margin-left: 0px;
+
   & .stakes {
     grid-area: stakes;
   }
 
   &::before {
     content: '';
-    min-width: 100%;
+    min-width: calc(100% - 30px);
```

Comment:
If you use one of the exiting variables instead of the hard coded `30px`, could can be sure this will remain aligned with the other components later on if we decide to change the box spacings.